.drone.yml

```diff
@@ -20,7 +20,7 @@ steps:
       REALM_API_URL:
         from_secret: REALM_API_URL
       REALM_SEARCH_URL:
-        from_secret: REALM_SEARCH_URL
```

Comment:
Just curious, is the intention to update the existing realm search function with the contents on the staging one? If so, just want to be sure that we set this back.

Reply:
@mcolella14 I asked to add this since we don't have a staging version.